src/modules/fancyzones/lib/FancyZonesDataTypes.cpp

```diff
@@ -0,0 +1,102 @@
+#include "pch.h"
+#include "util.h"
+
+#include "FancyZonesDataTypes.h"
+
+namespace
+{
+    // From Settings.cs
+    constexpr int c_focusModelId = 0xFFFF;
+    constexpr int c_rowsModelId = 0xFFFE;
+    constexpr int c_columnsModelId = 0xFFFD;
+    constexpr int c_gridModelId = 0xFFFC;
+    constexpr int c_priorityGridModelId = 0xFFFB;
+    constexpr int c_blankCustomModelId = 0xFFFA;
+}
+
+namespace FancyZonesDataTypes
+{
+    std::wstring TypeToString(ZoneSetLayoutType type)
+    {
+        switch (type)
+        {
+        case ZoneSetLayoutType::Blank:
+            return L"blank";
+        case ZoneSetLayoutType::Focus:
+            return L"focus";
+        case ZoneSetLayoutType::Columns:
+            return L"columns";
+        case ZoneSetLayoutType::Rows:
+            return L"rows";
+        case ZoneSetLayoutType::Grid:
+            return L"grid";
+        case ZoneSetLayoutType::PriorityGrid:
+            return L"priority-grid";
+        case ZoneSetLayoutType::Custom:
+            return L"custom";
+        default:
+            return L"TypeToString_ERROR";
+        }
+    }
+
+    ZoneSetLayoutType TypeFromString(const std::wstring& typeStr)
+    {
+        if (typeStr == L"focus")
```

Comment:
Replace all strings literals with const strings and explicitly mark them as non-localizable.

Reply:
This is marked as resolved, but the literals are still there.